dedupe/core.py

```diff
@@ -266,7 +266,7 @@ def fillQueue(queue, iterable, stop_signals) :
                 last_rate = current_rate
                 n_records = 0
                 t0 = time.clock()
-                
+                chunk.clear()
```

Comment:
why do you want to do this ever 10th time. shouldn't you do this every time (bring in the clear one indent level)